src/qt/walletcontroller.cpp

```diff
@@ -206,7 +206,7 @@ void CreateWalletActivity::askPassphrase()
 
 void CreateWalletActivity::createWallet()
 {
-    showProgressDialog(tr("Creating Wallet <b>%1</b>...").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
+    showProgressDialog(tr("<div align='center'>Creating Wallet</div><hr><wbr /><div align='center'><b>%1</b></div>").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
```

Comment:
Why mixing HTML-style `<hr>` with XHTML-style `<wbr />` in the same line?

Reply:
A ```<br />```  was causing too much extra space in the form. 

![Screen Shot 2020-01-21 at 5 26 41 PM](https://user-images.githubusercontent.com/152159/72854293-e80f8380-3c81-11ea-9346-987cb65ab295.png)

Reply:
I'm talking about `<hr>` vs `<hr />` and `<wbr>` vs `<wbr />`.

Reply:
Thanks for the feedback! 👍 

![Screen Shot 2020-01-21 at 9 25 32 PM](https://user-images.githubusercontent.com/152159/72860568-30846c80-3c95-11ea-9155-564e18a5e082.png)


![Screen Shot 2020-01-21 at 9 25 49 PM 1](https://user-images.githubusercontent.com/152159/72860574-35e1b700-3c95-11ea-99d0-ab0aee3b9292.png)


![Screen Shot 2020-01-21 at 9 26 51 PM](https://user-images.githubusercontent.com/152159/72860592-41cd7900-3c95-11ea-9c4e-33dc6bf58777.png)

Old behavior:
![](https://user-images.githubusercontent.com/152159/72851574-03769080-3c7a-11ea-861e-783748c1ffce.png)